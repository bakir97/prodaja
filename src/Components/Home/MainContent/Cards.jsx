import React from "react";
import Card from "../../../Layout/Card/Card";
import { Grid } from "@material-ui/core";
const Cards = ({ proizvodi, layout, history }) => {
  return proizvodi.map(({ naslov, slike, cijena, tezina, _id }) => (
    <Grid key={_id} item container justify="center" xs={layout}>
      <Card
        history={history}
        id={_id}
        naslov={naslov}
        slika={slike[0]}
        cijena={cijena}
        tezina={tezina}
      />
    </Grid>
  ));
};

export default Cards;
