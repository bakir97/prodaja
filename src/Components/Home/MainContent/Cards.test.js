import Cards from "./Cards";
describe("Cards", () => {
  const proizvodi = [
    {
      naslov: "test",
      slike: [1, 2],
      _id: 1,
      cijena: 1,
      tezina: 1
    },
    {
      naslov: "test2",
      slike: [1, 2],
      _id: 2,
      cijena: 2,
      tezina: 2
    }
  ];
  const wrapper = shallow(<Cards proizvodi={proizvodi} />);
  it("should render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
