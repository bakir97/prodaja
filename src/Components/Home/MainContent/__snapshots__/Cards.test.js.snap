// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards should render 1`] = `
Array [
  <WithStyles(Grid)
    container={true}
    item={true}
    justify="center"
    key="1"
  >
    <WithStyles(MyCard)
      cijena={1}
      id={1}
      naslov="test"
      slika={1}
      tezina={1}
    />
  </WithStyles(Grid)>,
  <WithStyles(Grid)
    container={true}
    item={true}
    justify="center"
    key="2"
  >
    <WithStyles(MyCard)
      cijena={2}
      id={2}
      naslov="test2"
      slika={1}
      tezina={2}
    />
  </WithStyles(Grid)>,
]
`;
